src/Microsoft.ML.Data/Transforms/HashTransform.cs

```diff
@@ -1258,5 +1294,63 @@ public override void Process()
             }
         }
     }
+
+    public sealed class HashConverter : IEstimator<HashConverterTransformer>
```

Comment:
>HashConverter [](start = 24, length = 13)

I would prefer if we could maintain a policy that estimators tend to have "estimator" at the end of their names, and transformers tend to have "transformer" at the end of their names. Otherwise picking out what is which will be utterly bewildering for users...

Reply:
Plus of course, again, I think "Hash converter" is not the best name ever.

---
In reply to: [218940371](https://github.com/dotnet/machinelearning/pull/944#discussion_r218940371) [](ancestors = 218940371)

Reply:
Well, I got sign off from Pete, with this Converter, I guess I can sneakily rename them back to estimators to get sign off from you :)

---
In reply to: [218940589](https://github.com/dotnet/machinelearning/pull/944#discussion_r218940589) [](ancestors = 218940589,218940371)

Reply:
I could see that!

---
In reply to: [218943451](https://github.com/dotnet/machinelearning/pull/944#discussion_r218943451) [](ancestors = 218943451,218940589,218940371)